src/main/integrations/last-fm/index.ts

```diff
@@ -40,12 +40,30 @@ export default class LastFM implements IIntegration {
     this.lastfmDetails.token = await this.createToken();
     this.saveSettings();
 
-    shell.openExternal(
-      `https://www.last.fm/api/auth/` + `?api_key=${encodeURIComponent(this.lastfmDetails.api_key)}` + `&token=${encodeURIComponent(this.lastfmDetails.token)}`
-    );
+    let authWindow = new BrowserWindow({
```

Comment:
Curious to the inclusion of using a Browser window and not using the user's own Browser which will have their Login/Password already stored and ready.
I feel that from both a User perspective this also seems like a better "secure" option considering it's not being processed through YTMD.

Reply:
The reason for this was to track when the user closed the browser after allowing access for step `2.3. Request authorization from the user` which must be done before we fetch a user session for scrobbling.  
Before this change the token was returning an error as it wasn't authorized to a user.
I admit i did this pretty hastily, but it works. I'm open to input but the current code is simply broken and does things out of order.

Reply:
Ok I was able to revert this change once I found the **real** bug, this is still a little janky and I think requires the user to toggle the setting off and then back:

If the token fails: get a new token and reauth them, but a new session isn't created until the integration switch is set off and then enabled again

Reply:
Session is grabbed on next time a song is played, so it's not entirely ignored and a problem.
I do want to add a `refresh` button to the Settings page, but I need to do some exposing of the APIs from the integration to the settings.